src/sysrepo.c

```diff
@@ -5840,7 +5840,7 @@ sr_module_change_subscribe_enable(sr_session_ctx_t *session, struct sr_mod_info_
     if ((err_info = sr_modinfo_add(ly_mod, NULL, 0, 0, mod_info))) {
         goto cleanup;
     }
-    if ((err_info = sr_modinfo_consolidate(mod_info, SR_LOCK_READ, SR_MI_PERM_NO, session, 0, 0, SR_OPER_NO_SUBS))) {
+    if ((err_info = sr_modinfo_consolidate(mod_info, SR_LOCK_READ, SR_MI_PERM_NO, session, 0, 0, SR_OPER_NO_SUBS | SR_OPER_WITH_NP_CONT))) {
         goto cleanup;
     }
 
```

Comment:
switching below to sr_lyd_new_implicit_module() caused test_oper_pull to fail, and I couldn't make sense of why.